docs/src/Avatar.doc.js

```diff
@@ -45,6 +45,11 @@ card(
         type: 'boolean',
         defaultValue: false,
       },
+      {
+        name: 'icon',
+        type: '"check-circle" | "pinterest"',
+        defaultValue: false,
```

Comment:
default value should be `check-circle` here and i would add a `description` that says "This is the icon shown only when `verified=true`"